build/tasks/languages.js

```diff
@@ -0,0 +1,31 @@
+module.exports = function(grunt) {
+  grunt.registerTask('check-translations', 'Check that translations are up to date', function(){
+    const source = require('../../lang/en.json');
+    let doc = grunt.file.read('docs/translations-needed.md');
+    let table = '|---|---|\n';
```

Comment:
might be better to build out the table using something like https://github.com/wooorm/markdown-table